bin/import.ml

```diff
@@ -51,12 +51,20 @@ end = struct
     Console.Status_line.set
       (Live
          (fun () ->
+           let open Pp.O in
+           let waiting = Pp.verbatim ", waiting for filesystem changes..." in
            match Fiber.Svar.read Build_system.state with
-           | Initializing
-           | Interrupting_current_build
-           | Build_finished_with_success_and_waiting
+           | Initializing -> Pp.verbatim "Initializing..."
+           | Build_finished_with_success_and_waiting ->
+             Pp.tag User_message.Style.Success (Pp.verbatim "Success")
+             ++ waiting
            | Build_finished_with_failure_and_waiting ->
-             Pp.nop
+             Pp.tag User_message.Style.Error (Pp.verbatim "Had errors")
+             ++ waiting
+           | Interrupting_current_build ->
+             Pp.tag User_message.Style.Error
+               (Pp.verbatim "Source files changed")
```

Comment:
@jeremiedimino it seems like the semantics of `Fiber.Svar.t` mean that some notifications might be missed. In particular, it's not a guarantee that the UI will observe the `Interrupting_current_build` event. In practice, I suppose it's not a big deal as users only care about the current results. However, I think they would prefer to always see the delimiters before/after builds.

Reply:
We have to choose between a signal and an event semantic, and we choose a signal semantic, so it's expected we don't see all the state changes.

Why do you think users would want to see the delimiters? If it's to know whether a new build has started, then there is a simple solution for that: we can include the build number in the state.

Reply:
Ah wait, I didn't realise this comment was for the console printer. For the console, I agree that we want to see the new build delimiters, given that it is a continuous output. I remember thinking about this when I did this change, and I convinced myself that this was still the case because we call `Console.Status_line.refresh` on every iteration (via the `Tick` event handler).

Reply:
But now that I write this down, it feels a bit fragile. Maybe we should add a primitive `Fiber.Svar.changes : 'a t -> 'a Fiber.Stream.In.t`?

Reply:
Scratch that, looking again at the code, it looks all good to me because the `*** New build ***` messages are printed separately. The printing code I moved was all for temporary messages.

Reply:
Hmm, some tests are failing. I can see the issue: `Console.Status_line.set (Constant _)` always print something, but `Console.Status_line.set (Live _)` prints nothing if the output is not a terminal

Reply:
It seems complicated to fix this. I liked this refactoring as it centralised the logic to render the status line and removed some duplication between the build system state and scheduler events, but it seems complicated to get back the old behaviour. For now you should just drop my commit.